doc/README.md

```diff
@@ -36,8 +36,10 @@ MarketMakerProxy is an example implementation of a EIP-1271 compatable contract
 
 # Strategy Contracts
 
+There are multiple strategy contracts in the Tokenlon protocol with different trading mechanisms. A user can navigate to each strategy contract thorugh dedicated routing function in `UserProxy` contract. Each routing function takes one parameter `bytes _payload` which is later then forwarded to a specific strategy contract.
```

Comment:
```suggestion
There are multiple strategy contracts in the Tokenlon protocol with different trading mechanisms. A user can navigate to each strategy contract through dedicated routing function in `UserProxy` contract. Each routing function takes one parameter `bytes _payload` which is later then forwarded to a specific strategy contract.
```

Reply:
I think we can leave the detail (`_payload` param) to UserProxy of strategy contracts.
```suggestion
There are multiple strategy contracts in the Tokenlon protocol with different trading mechanisms. A user can navigate to each strategy contract thorugh dedicated routing function in `UserProxy` contract.
```